src/libp2p/service.rs

```diff
@@ -548,6 +549,21 @@ async fn handle_network_message(
                         warn!("Failed to disconnect from a peer");
                     }
                 }
+                NetRPCMethods::AgentVersion(response_channel, peer_id) => {
+                    let agent_version = swarm
+                        .behaviour()
+                        .peer_info(&peer_id)
+                        .map(|info| {
+                            info.agent_version
+                                .clone()
+                                .unwrap_or_else(|| "item not found".to_string())
```

Comment:
I guess we can omit `unwrap_or_else` here if the channel takes `Option<String>`, and let the caller (which is more on UX side) to decide the fallback string

Reply:
Let's keep the fallback the same as in Lotus.